docs/getting_started/install_manually.md

```diff
@@ -385,6 +393,14 @@ If Composer asks you for your token, you must log in to your GitHub account and
 
 Please note that a clean install of eZ Platform doesn’t include the DemoBundle anymore.
 
+!!! enterprise
+
+    j. Enable Date-based Publisher
+
+    To enable delayed publishing of Content using the Date-based publisher, run the following line:
+
+    `php app/console ezstudio:scheduled:publish --quiet --env=prod`
```

Comment:
It is not as strigt forward as this.

The command needs to be run on a regular basis. If you need minute resolution ( ie possibility to publish something at 10:32, and next article at 10:33) you need to run this once per minute. If a 5 minute resolution is fine, then run it every 5 minute..

The usual way running commands regulary on linux/unix is using a crontab (I don't know about Mac)
The second problem is to decide as which user the crontab should run ( user stuff is covered a bit later in this doc ).
It can run either as the web server user, or as the CLI user.

Assuming the web server user data is www-data, the commands to add this script in a cron is (run the commands as root):

```
    # for one-minute resolution
    echo '* * * * * cd [path-to-ezplatform]; php app/console ezstudio:scheduled:publish --quiet --env=prod' > ezp_cron.txt
    # for 5-minutes resolution, do instead
    echo '*/5 * * * * cd [path-to-ezplatform]; php app/console ezstudio:scheduled:publish --quiet --env=prod' > ezp_cron.txt

    # Next, append the new cron to user's crontab without destroying existing crons
    crontab -u www-data -l|cat - new_cron.txt | crontab -u www-data -

    # remove tmp file
    rm new_cron.txt
```

Reply:
Thanks for the explanation, extended the description according to your comment

Reply:
Note: I think the intention from studio team is that it would be run via:
https://github.com/ezsystems/ezstudio-cron

At least the command here is registered for it:
https://github.com/ezsystems/date-based-publisher/search?utf8=%E2%9C%93&q=ezpublish.cron.job&type=